### PR TITLE
Disable Initial Prompt Task for en and es Locales

### PR DESCRIPTION
**Pull request by hzj5790**
To help clear the prompt backlog, as per https://github.com/LAION-AI/Open-Assistant/issues/1659, and Andreas's suggestions in https://github.com/LAION-AI/Open-Assistant/pull/1824, this disables the initial-prompt task for en and es locales.

### CODE REVIEW

**Comment by andreaskoepf** [comment]
Oh, why in the frontend? It would be ~10 lines total in the backend to add this?

**Comment by notmd** [comment]
> Oh, why in the frontend? It would be ~10 lines total in the backend to add this?

Agree, this should be done in backend, maybe via env variable or something

File: backend/oasst_backend/tree_manager.py [hunk 1]
```diff
@@ -366,8 +367,14 @@ def determine_task_availability(self, lang: str) -> dict[protocol_schema.TaskReq
             lang = "en"
             logger.warning("Task availability request without lang tag received, assuming lang='en'.")
 
+        init_prompt_disabled_langs = os.getenv("INIT_PROMPT_DISABLED_LANGS", "none")
```
Comment (andreaskoepf) [inline backend/oasst_backend/tree_manager.py:370]: Could you move this to the [TreeManagerConfiguration](https://github.com/LAION-AI/Open-Assistant/blob/main/backend/oasst_backend/config.py)? The setting then still can be set via env-variable and this setting would be handled in the same way all other TreeManager settings are handled.

**Comment by hzj5790** [comment]
Closing as https://github.com/LAION-AI/Open-Assistant/pull/1889 seems to be a better fix. I'm sorry for consuming a fair bit of your time @andreaskoepf.

**Comment by andreaskoepf** [comment]
> Closing as #1889 seems to be a better fix. I'm sorry for consuming a fair bit of your time @andreaskoepf, only to arrive at this point.

We can add your manual parameter setting too … it is a minimal change and would be an additional setting.

**Comment by andreaskoepf** [comment]
This PR still tries to remove the latest changes in tree_manager.py ...

**Comment by olliestanley** [comment]
@andreaskoepf I made those small changes to this PR so hopefully it is now compatible with latest `main` if you want to merge

**Review by andreaskoepf — APPROVED** [review]
ok, let's get that option in.

**Pull request merged**